cr.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  Netguru iOS code review task
+//
+
+class paymentViewController: UIViewController {
```

Comment:
I would follow convention to start class name with capital letter.
If we don't want to class to be inherited we can mark it as final.